### DISCUSSION TITLE
Reflect many to many relationship on both sides.

### DISCUSSION THREAD

**Discussion by development-vargamarcel** [directus/directus#12432] [Archive]
example: **tableA,tableB,tableA_tableB**

In this scenario i would like when i create a many to many relationship,to be able to:
 **1.  view and add tableB rows relating to tableA from tableA view
 2.  view and add tableA rows relating to tableB from tableB view**

In the current state i can have one of the above but not both. (**1.** with  tableB_tableA , **2** with  tableA_tableB )

Creating both tableB_tableA and  tableA_tableB would give me the right ui but would make no sense for obvious reasons.

I see this as a bug but i guess this is intetional.is there a reason the implementation is as is now?

**Comment by rijkvanzanten** [comment]
> In the current state i can have one of the above but not both.

That's not true! You can have both still 👍🏻 

By going into the _Advanced Field Creation_, you can configure exactly what relational fields to use:

<img width="794" alt="CleanShot 2022-03-28 at 17 44 19@2x" src="https://user-images.githubusercontent.com/9141017/160492464-94232e5f-2497-4389-885a-b9ccaa557293.png">

<img width="632" alt="CleanShot 2022-03-28 at 17 44 59@2x" src="https://user-images.githubusercontent.com/9141017/160492518-2baa14b0-ed71-40f8-a5b3-d149f7a842cd.png">

**Reply by development-vargamarcel** [reply]
Thank you very much. And I am sorry for not diving deeper. The UI is straightforward enough, my bad.

**Reply by rijkvanzanten** [reply]
No worries! There's a lot of power hidden away in the platform 🙂 Can't expect everybody to know everything right away!

That does remind me, is there anything we could've done better to make sure you knew about this earlier on? Sounds like it could be improved in the docs for example (cc @erondpowell)
